pkg/apply/basic_printer.go

```diff
@@ -132,3 +133,12 @@ func getName(obj runtime.Object) string {
 func resourceIDToString(gk schema.GroupKind, name string) string {
 	return fmt.Sprintf("%s/%s", strings.ToLower(gk.String()), name)
 }
+
+func (b *BasicPrinter) getPrintFunc(preview bool) func(message string) {
+	return func(message string) {
+		if preview {
+			message += " (preview)"
+		}
+		fmt.Fprintf(b.IOStreams.Out, message+"\n")
```

Comment:
We could use the `Fprint` function here since we are not doing any interpolation.